### PR TITLE
[64] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by Marienoir**
## Task: `Styling the Product Details Page using CSS`
- `I styled  the Product Details Page using CSS`

### CODE REVIEW

File: submissions/maryOkosun/css/product.css [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+		width: 80%;
+	}
+	footer button {
+		padding: 2px;
+		margin-left: 3px;
+		margin-right: 1px;
+	}
+}
+
+
```
Comment (Ireti203) [inline submissions/maryOkosun/css/product.css:276]: Remove extra whitespace

File: submissions/maryOkosun/css/product.css [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+		height: 260px;
+		width: 90%;
+		border:1px solid white;
+		display: block;
+		margin: 20px;
+		margin-bottom: 20px;
+		background-color: white;
+	}
+	.empty-box img{
+
```
Comment (Ireti203) [inline submissions/maryOkosun/css/product.css:344]: Remove whitespace.

File: submissions/maryOkosun/css/product.css [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+		height: 180px;
+		padding-left: 70px;
+		padding-top: 10px;
+
+	}
+	.empty-box p{
+		text-align: center;
+		font-weight: bold;
+	}
+
```
Comment (Ireti203) [inline submissions/maryOkosun/css/product.css:354]: Please use a consistent formatting.

**Review by Ireti203 — COMMENTED** [review]
Weldone@Marienoir

**Comment by Ireti203** [comment]
Your images (under related products) looks tilted to the extreme right on medium screen size. Can you make it remain at the center of the box just like you have on the small screen and large screen sizes.

File: submissions/maryOkosun/css/product.css [hunk 4]
```diff
@@ -0,0 +1,390 @@
+*{
+	box-sizing: border-box;
+	font-family: 'Montserrat', sans-serif;
+}
+body{
+	margin: 0 auto;
+	font-size: 16px;
+	padding: 0;
+	height: 152vh;
+
```
[the remark this answers is not in the document]
Comment (Marienoir) [reply submissions/maryOkosun/css/product.css]: Thanks. 
I have made the necessary corrections.

**Comment by Marienoir** [comment]
tab modified

**Review by adaugochi — APPROVED** [review]

**Pull request merged**